pkg/echo/echo.go

```diff
@@ -45,20 +47,28 @@ func echoHandler(w http.ResponseWriter, r *http.Request) {
 
 	// print the data sent to the echo server to the console
 
-	color.Green("-- %s %s -> created %d\n", r.Method, r.URL.Path, http.StatusCreated)
-	fmt.Printf("received %d readings:\n", len(payload.DataReadings))
+	if Compact {
+		fmt.Printf("-- %s %s -> created %d\n", r.Method, r.URL.Path, http.StatusCreated)
+		fmt.Printf("received %d readings:\n", len(payload.DataReadings))
+		for _, r := range payload.DataReadings {
+			fmt.Printf("%+v\n", r)
+		}
+	} else {
+		color.Green("-- %s %s -> created %d\n", r.Method, r.URL.Path, http.StatusCreated)
```

Comment:
I think it's good :+1:  we could also have a function specifically for print in colours and one to printed the compacted output, what do you think?